src/TicketDatabase.jsx

```diff
@@ -0,0 +1,11 @@
+import { includes } from 'lodash';
```

Comment:
Not something we have specified, but I'd prefer files that don't include JSX code to simply have the extension `.js`